tests/show.rs

```diff
@@ -43,6 +43,7 @@ fn fuelup_show_latest() -> Result<()> {
                 - forc-fmt : 0.1.0
                 - forc-lsp : 0.1.0
                 - forc-migrate : 0.1.0
+                - forc-node : not found
```

Comment:
nit: would expect a version value; however unsure how this works

Reply:
I said the same thing :) see this comment https://github.com/FuelLabs/fuelup/pull/720#issuecomment-2840449777

Reply:
https://github.com/FuelLabs/fuelup/pull/720#issuecomment-2840449777